src/main/resources/org/broadinstitute/hellbender/utils/helpTemplates/metrics.template.html

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
This is just copied from picard I assume?

Reply:
Its copied from the Picard one that I made up when I added the metrics ;-). I removed the group in from it though, since thats not usually included in the detail pages and for some reason I left it in the Picard one (which is not really used anyway).